### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mariaschett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,80 +7,62 @@
...
-in order to establish that a legitimate version of the Oak trusted runtime is
-indeed running server side, before actually sending any data to it.
-
-Further information is included in the following documents:
-
-- [Oak Development](docs/development.md) covers practical steps for getting a
-  development Oak system up and running.
-- [Oak Functions](oak_functions/README.md) presents our computing platform for
-  developing stateless applications in a privacy preserving way.
+To do so, Oak relies on running _Oak Trusted Enclave Applications_ in a
```
[the remark this answers is not in the document]
Comment (tiziano88) [reply README.md:13]: Also I would keep the sentence singular (talking about a single application)

File: README.md [hunk 2]
```diff
@@ -7,80 +7,62 @@
...
+[Trusted Execution Environment (TEE)](https://en.wikipedia.org/wiki/Trusted_execution_environment).
+An example of a Trusted Enclave Application is
+[Oak Functions](/oak_functions/README.md). The Trusted Enclave Application can
+provide the client cryptographically attested evidence of the security
```
[the remark this answers is not in the document]
Comment (tiziano88) [reply README.md:17]: +1, eventually I would like to focus the Oak narrative on transparency first, and then show how transparency can be used to show security or privacy guarantees.

File: README.md [hunk 3]
```diff
@@ -7,80 +7,62 @@
...
+properties by [Remote Attestation](./docs/remote-attestation.md). Together with
+[Transparent Release](https://github.com/project-oak/transparent-release) this
+binds the open-source source code to the remotely attested binary running inside
+the TEE. In order to feasibly review all the source code running inside the TEE,
+and minimize our trusted computing base, Oak provides the following
+infrastructure: [stage 0](/stage0/),
+[Oak Restricted Kernel](/oak_restricted_kernel/) and a controlled communications
+interfaces---the [Oak Comms Channel](/oak_channel/) and [microRPC](/micro_rpc/).
```
Comment (tiziano88) [inline README.md:25]: I don't think the `---` is rendered correctly; anyways, probably a bullet list here may be better (typographically)

File: README.md [hunk 4]
```diff
@@ -7,80 +7,62 @@
...
-  - Operating System (kernel, drivers, libraries, applications)
   - Platform Provider
-  - Untrusted Business Logic Authors
+  - Host Operating System (kernel, drivers, libraries, applications)
+  - Hypervisor / VMM
 - **trusted-but-verifiable**:
-  - Oak Trusted Runtime Authors (their actions are verifiable via
+  - Oak Infrastructure Authors
+  - Trusted Enclave Application Authors
+  - (their actions are verifiable via
```
Comment (tiziano88) [inline README.md:50]: This should be nested under the previous one, I think

**Review by tiziano88 — APPROVED** [review]
Thanks Maria

File: README.md [hunk 5]
```diff
@@ -7,80 +7,62 @@
 [![Build Status](https://img.shields.io/github/actions/workflow/status/project-oak/oak/ci.yaml?branch=main&style=for-the-badge)](https://github.com/project-oak/oak/actions/workflows/ci.yaml?query=branch%3Amain)
 [![Docs](https://img.shields.io/badge/docs-rust-brightgreen?style=for-the-badge)](https://project-oak.github.io/oak)
 
-The goal of Project Oak is to create a specification and a reference
-implementation for the secure transfer, storage and processing of data.
+The goal of Project Oak is to provide infrastructure to transfer, store and
+process data in a secure and transparent way.
```
Comment (k-naliuka) [inline README.md:11]: Maybe explicitly mention "sensitive user data" to highlight the motivation

**Review by k-naliuka — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (mariaschett) [reply README.md]: Good point, done.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariaschett) [reply README.md]: Both good points and done!

File: README.md (hunk 3 above)
Comment (mariaschett) [reply README.md]: Replaced by i.e.,

File: README.md (hunk 4 above)
Comment (mariaschett) [reply README.md]: Rephrased to "trusted-but-transparent". Should the "Trusted Enclave Application Authors" be on there? I think so. Other opinions?

**Review by mariaschett — COMMENTED** [review]
Thank you for @k-naliuka & @conradgrobler for peer-writing this! 

And, of course, thanks for the reviews!

File: README.md [hunk 6]
```diff
@@ -7,80 +7,62 @@
...
-- [Oak Development](docs/development.md) covers practical steps for getting a
-  development Oak system up and running.
-- [Oak Functions](oak_functions/README.md) presents our computing platform for
-  developing stateless applications in a privacy preserving way.
+To do so, Oak relies on running _Oak Trusted Enclave Applications_ in a
+[Trusted Execution Environment (TEE)](https://en.wikipedia.org/wiki/Trusted_execution_environment).
+An example of a Trusted Enclave Application is
+[Oak Functions](/oak_functions/README.md). The Trusted Enclave Application can
+provide the client cryptographically attested evidence of the security
+properties by [Remote Attestation](./docs/remote-attestation.md). Together with
```
[the remark this answers is not in the document]
Comment (conradgrobler) [reply README.md]: Yes, thanks.

**Pull request merged**